tests/test_vcs_update_triggers_continents.py

```diff
@@ -0,0 +1,9 @@
+import basevcstest
+
+class TestVCSTextsExtents(basevcstest.VCSBaseTest):
+    def testUpdateDoesNotTriggerContinents(self):
+        s = self.clt["clt"][0]
+        self.x.plot(s, continents=0)
```

Comment:
Interesting, I guess while I knew you could disable the continents somehow, I never knew this is how you did it.